react/features/toolbar/defaultToolbarButtons.js

```diff
@@ -0,0 +1,334 @@
+/* @flow */
+
+import React from 'react';
+
+import UIEvents from '../../../service/UI/UIEvents';
+
+declare var APP: Object;
+declare var config: Object;
+declare var JitsiMeetJS: Object;
+
+/**
+ * Shows SIP number dialog.
+ *
+ * @returns {void}
+ */
+function _showSIPNumberInput() {
+    const defaultNumber = config.defaultSipNumber || '';
+    const titleKey = 'dialog.sipMsg';
+    const msgString = `
+            <input class="input-control"
+                   name="sipNumber" type="text"
+                   value="${defaultNumber}" autofocus>`;
+
+    APP.UI.messageHandler.openTwoButtonDialog({
+        titleKey,
+        msgString,
+        leftButtonKey: 'dialog.Dial',
+
+        // eslint-disable-next-line max-params
+        submitFunction: (event, value, message, formValues) => {
+            const { sipNumber } = formValues;
+
+            if (value && sipNumber) {
+                APP.UI.emitEvent(UIEvents.SIP_DIAL, sipNumber);
+            }
+        },
+        focus: ':input:first'
+    });
+}
+
+/**
+ * All toolbar buttons' descriptions.
+ */
+export default {
+    microphone: {
```

Comment:
Now that these look like simple object property names, it should be obvious that our coding style requires:
1. jsdocs on everything. I believe at least the top-level objects which describe buttons should have jsdocs.
2. Alphabetical sorting. I don't see any advantage to the random order of button descriptors.